external/implementations/samlconf-openam-impl/src/main/java/org/codice/compliance/saml/plugin/openam/OpenAMDataParser.java

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright (c) 2018 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+package org.codice.compliance.saml.plugin.openam;
```

Comment:
📘 Because OpenAM's login page is configurable (https://github.com/OpenIdentityPlatform/OpenAM/wiki/Write-a-custom-authentication-module) This file is used to parse an XML file that has the configurable information in it.